### PR TITLE
squid module init setup

### PR DESCRIPTION
**Pull request by BKS1989**

### CODE REVIEW

**Comment by BKS1989** [comment]
grok pattern of access logs

File: filebeat/module/squid/access/_meta/fields.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+- key: access
+  type: group
+  description: >
+    squid Module
+  fields:
+    - name: duration
```
Comment (ruflin) [inline filebeat/module/squid/access/_meta/fields.yml]: This needs completion.

Comment (BKS1989) [reply filebeat/module/squid/access/_meta/fields.yml]: I have added test log

Comment (ruflin) [reply filebeat/module/squid/access/_meta/fields.yml]: Could you also update the description of the fields.

Looking at your ingest pipeline below it seems it does not put the fields into the same fields as defined here. You must be aware that all the fields should be prefixed by `squid.access` if they don't overlap with some global fields.

**Comment by Randy-312** [comment]
Possible for this to be moved forward?

**Comment by blakerouse** [comment]
@BKS1989 We are in the process of cleanup some old PR's, noticed this hasn't had any activity in a while. Are you still interested in getting this merged? Would you be able to rebase with latest master?

**Comment by opoplawski** [comment]
I have a question about squid parsing status.  I notice that commit 6d0dc6260deda0df046c59d601885335f6c2c071 added a squid module under the x-pack tree.  Would beats still be amenable to an updated version of this pull request that would be freely available?  Or are we stuck with x-pack only now?

**Pull request closed without merging**